apps/fluent-tester/src/E2E/Menu/pages/MenuPageObject.ts

```diff
@@ -23,25 +22,19 @@ class MenuPageObject extends BasePage {
   /**************** UI Element Interaction Methods ******************/
   /******************************************************************/
   async didMenuOpen(): Promise<boolean> {
-    const callbackText = await By(MENU_ON_OPEN);
-    await browser.waitUntil(async () => await callbackText.isDisplayed(), {
+    await browser.waitUntil(async () => await this.menuIsExpanded(), {
       timeout: this.waitForUiEvent,
       timeoutMsg: 'The Menu did not open.',
       interval: 1000,
     });
 
-    return await callbackText.isDisplayed();
+    return await this.menuIsExpanded();
   }
 
-  async didMenuClose(): Promise<boolean> {
-    const callbackText = await By(MENU_ON_CLOSE);
-    await browser.waitUntil(async () => await callbackText.isDisplayed(), {
-      timeout: this.waitForUiEvent,
-      timeoutMsg: 'The Menu did not close.',
-      interval: 1000,
-    });
-
-    return await callbackText.isDisplayed();
+  async menuIsExpanded(): Promise<boolean> {
+    const expandState = await (await this._primaryComponent).getAttribute('ExpandCollapse.ExpandCollapseState');
+    console.log(expandState);
```

Comment:
would it be fair to say that we're testing that the ExpandCollapse state changes as-expected when we invoke the button, but we haven't actually verified that a menu appeared on-screen? Asking in part relative to the commented out code in Menu.spec.win.ts re: `TestID for Callouts aren't propagating, so this test is failing`

Reply:
This is a fair assessment. I've added a line of code that also tests if the primary menu item is also displayed in the same method.

Reply:
The accessibility tests also open the menu and query for a menu item, so we verify the menu appears there. And the line of code Larry added does the same.

@lawrencewin Perhaps we create a new accessibility test to check for the ExpandCollapse state (since it is the accessibility state). And then in the functional testing block, we only check if the menu is open by determining if the menu item is visible or not.

This way, we don't have 1 test checking for two separate conditions (accessibility state AND menu item visibility).

We want these tests to be as specific as possible. Right now, if this test fails, the developer has no idea if it failed because the ExpandCollapse state was wrong or if the menu item wasn't displayed.

Does that make sense?

Reply:
@samuelfreiberg ahh yeah my comment was mostly directed at clarifying this function vs the test as a whole, I articulated that poorly.  I'm not totally sure of all interpretations I made while reading this PR, can you help clarifying for me for me how the `'Validate OnOpenChange() callback was fired -> Type "SPACE"'` test queries the menu item?